packages/@glimmer/core/src/references.ts

```diff
@@ -229,26 +230,12 @@ export class UpdatableReference<T> extends ComponentPathReference<T> {
 }
 
 export class ConditionalReference extends GlimmerConditionalReference {
-  static create(reference: PathReference<any>) {
+  static create(reference: PathReference<boolean>): ConditionalReference | PrimitiveReference<boolean> {
     if (isConst(reference)) {
-      let value = reference.value();
+      const value = reference.value();
       return PrimitiveReference.create(value);
     }
 
     return new GlimmerConditionalReference(reference);
   }
 }
-
-export class TemplateOnlyComponentDebugReference extends ConstReference<void> {
-  constructor(protected name: string) {
-    super(undefined);
-  }
-
-  get(propertyKey: string): PathReference<unknown> {
-    throw new Error(
```

Comment:
Is this built-in now?

Reply:
Yes, but there have been more changes after the last VM upgrade in Ember. The plan is to followup with another PR to upgrade the VM again here, and rewrite some more things and move them upstream.